ddtrace/tracer/option.go

```diff
@@ -1001,7 +997,7 @@ func WithLogger(logger Logger) StartOption {
 // FinishOption.
 func WithDebugStack(enabled bool) StartOption {
 	return func(c *config) {
-		c.noDebugStack = !enabled
+		c.internalConfig.SetDebugStack(enabled, telemetry.OriginCode)
```

Comment:
Just for clarity + documentation purposes: this new config now says `true` if we DO want the debug stack, whereas before we said `true` if we did NOT want the debug stack? 😵 💫